sentry-ruby/lib/sentry-ruby.rb

```diff
@@ -92,6 +99,14 @@ def capture_message(message, **options, &block)
       get_current_hub.capture_message(message, **options, &block)
     end
 
+    def start_span(**options)
+      get_current_hub.start_span(**options)
+    end
```

Comment:
I would remove this function, we don't want to have too much surface.
Also, start_span alone shouldn't exist, this should only work on a transaction with `start_child`

Reply:
I just check, in JS it's still there but not used so it's deprecated.